packages/site-demo/content/product/components/thumbnail/index.mdx

```diff
@@ -34,9 +34,26 @@ Thumbnails come in two variants: `squared` and `rounded`
 
 <DemoBlock orientation="horizontal" demo="combined" />
 
-## Loading and error states
+## Loading skeleton
 
-<DemoBlock orientation="horizontal" demo="loading-error" withThemeSwitcher />
+<DemoBlock orientation="horizontal" demo="loading" withThemeSwitcher />
+
+While loading the image, the thumbnail displays a loading skeleton but only if the thumbnail has sufficient
```

Comment:
We always use the plural for components.
And never write "you"

I'd re-write this part like that :

Thumbnails display a loading state while loading image. To display loading state, thumbnails require defined size.

For example: